core/src/main/kotlin/pages/contentNodeProperties.kt

```diff
@@ -8,9 +8,16 @@ class SimpleAttr(val extraKey: String, val extraValue: String) : ExtraProperty<C
 
     companion object {
         fun header(value: String) = SimpleAttr("data-togglable", value)
+        fun togglableTarget(value: String) = SimpleAttr("data-togglable", value)
     }
 }
 
+data class ContentTab(val text: ContentText, val overriddenToggleTarget: List<String>)
+class ExtraTabs(val tabs: List<ContentTab>) : ExtraProperty<ContentNode> {
+    companion object : ExtraProperty.Key<ContentNode, ExtraTabs>
+    override val key: ExtraProperty.Key<ContentNode, ExtraTabs> = ExtraTabs
+}
```

Comment:
I've been trying to understand what the general flow is, but I can't piece it all together :(

Very first questions when looking at the API:

1. What is `Extra` about these tabs? Meaning, if these tabs are extra, there must be `Non-Extra` tabs - what's the difference? In other words, what is the `Extra` word supposed to highlight? 
2. What is `TogglableTarget`? Since it's a string, it can be virtually anything, so I actually have no idea. I can assume it'll be a CSS class or something?
3. When do I need to override `TogglableTarget`, and how do I do it? Why can't I just remove the unwanted tab from extras instead of overriding?
4. How does it match text to content? If I have some `ContentTab("function")` - how does it know which content to hide under this tag, are there any rules?